### ISSUE TITLE
Windows not launching

### ISSUE THREAD

**Issue by knownothing12138** [upscayl/upscayl#346]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#faq-).
- [X] I have checked that this issue isn't addressed in any other issue or pull request.

### Describe the Bug

When launched，nothing happends, no window pop up, only a process in task manager shows the application is running.
I've download both V2.5.5 from github release and V2.5.1 from the website , all result the same, not launching，at least no window pop up

### To Reproduce

after install, double click or try to run admin

### Expected Behavior

nothing

### Version/Commit Hash

 V2.5.5 from github release and V2.5.1 from the website

### OS Specifics

CPU:Intel i5-10400F GPU: NVIDIA GTX 1650 OS version: windows 11 22H2 22621.1702

### Screenshots

![image](https://github.com/upscayl/upscayl/assets/45874900/66212203-2fcc-4a4a-a362-5309196a31d7)

**Comment by aaronliu0130** [comment]
Did you check if there was an administrator icon on the taskbar? Sometimes UAC isn’t brought to the front.

**Comment by knownothing12138** [comment]
When I double click the icon, there is no window pop up or UAC confirm or any thing.
When I try to open the app with administrator, I do need to confirm UAC, but there is no window pop up after

**Comment by aaronliu0130** [comment]
Weird, I can't reproduce this. Could it be blocked by your antivirus?

**Comment by NayamAmarshe** [comment]
Yeah, probably a bug in Windows because we haven't seen any such reports before.

**Comment by ehobson** [comment]
> Yeah, probably a bug in Windows because we haven't seen any such reports before.

No, I have exactly the same problem on Linux. Upscayl 2.8.1 does not open. I've tried both AppImage and Debian. It does absolutely nothing.

**Comment by aaronliu0130** [comment]
@ehobson That's definitely a separate issue. Does it work after installing libvips, gtk3, openjpeg2, nss or libasound.so? If so then we haven't fixed #462 yet.
